spec/features/admin/members/admin_search_member_spec.rb

```diff
@@ -20,9 +20,9 @@
     scenario "Search with non-matching data" do
       fill_in('Email', with: current_user.email)
       click_button('Search')
-      expect(page).to have_content("No users matched that search")
+      expect(page).to have_content("No users matched your search")
```

Comment:
Could do `"No users matched search: #{search_term} "` ?
That way the user has feedback on what they've searched for and can identify a typo if they made one rather than thinking their search was correct and the system doesn't work?
Just a thought.